storage/persistent_storage/persistent-storage-ocs.adoc

```diff
@@ -15,22 +15,40 @@ Red Hat OpenShift Container Storage provides its own documentation library. The
 |See the following Red Hat OpenShift Container Storage documentation:
 
 |What’s new, known issues, notable bug fixes, and Technology Previews
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.4/html/4.4_release_notes/[Red Hat OpenShift Container Storage 4.4 Release Notes]
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/4.5_release_notes/[Red Hat OpenShift Container Storage 4.5 Release Notes]
 
 |Supported workloads, layouts, hardware and software requirements, sizing and scaling recommendations
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.4/html/planning_your_deployment/index[Planning your Red Hat OpenShift Container Storage 4.4 deployment]
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/planning_your_deployment/index[Planning your Red Hat OpenShift Container Storage 4.5 deployment]
 
-|Deploying Red Hat OpenShift Container Storage 4.4 on an existing {product-title} cluster
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.4/html/deploying_openshift_container_storage/index[Deploying Red Hat OpenShift Container Storage 4.4]
+|Instructions on preparing to deploy when your environment is not directly connected to the internet
```

Comment:
In all these sections, I don't think it's necessary to say "Instructions on" so I would start with the gerund:

- Preparing to deploy...
- Deploying...
- Deploying and managing...
- Troubleshooting...